src/Components/Users/Users.module.css

```diff
@@ -41,3 +41,28 @@
   top: 30%;
   left: 0.5rem;
 }
+
+.btnactions{
+  display: flex;
+  align-items: center;
+  width: 150px; /* Adjust the width as needed */
+
+}
+
+.btnactions span {
+  margin-right: 5px; /* Adjust as needed */
+}
+
+.eyeIcon {
+  cursor: pointer;
+}
+.blockunblock {
+  width: 85px;
+  margin-right: 5px;
+  font-family: "Montserrat-Regular", Helvetica;
+  font-size: 16px;
+  border: none;
+}
+.blockunblock:focus,.eyeIcon:focus{
```

Comment:
@Rawan-Mohamed ,  you changed here the focus state of the icon, but what I meant is to nest the icon in a button, because a button by default can be pressed by clicking the spacebar or Enter, but this is not true with icons